ccmm_invenio/fixtures/vocabularies.yaml

```diff
@@ -0,0 +1,59 @@
+accessrights:
+  pid-type: v-ar
+  data-file: ccmm_access_rights.yaml
+
+resourceagentroletypes:
+  pid-type: v-agr
+  data-file: ccmm_agent_roles.yaml
+
+titletypes:
+  pid-type: v-tt
+  data-file: ccmm_alternate_title_types.yaml
+
+contributorsroles:
+  pid-type: v-ct
+  data-file: ccmm_contributor_types.yaml
+
+filetypes:
+  pid-type: v-ft
+  data-file: ccmm_file_types.yaml
+
+languages:
+  pid-type: v-lan
+  data-file: ccmm_languages.yaml
+
+licenses:
+  pid-type: v-lic
+  data-file: ccmm_licenses.yaml
+
+locationrelationtypes:
+  pid-type: v-lr
+  data-file: ccmm_location_relation_types.yaml
+
+relationtypes:
+  pid-type: v-rel
+  data-file: ccmm_relation_types.yaml
+
+resourcetypes:
+  pid-type: v-res
+  data-file: ccmm_resource_types.yaml
+
+subjectcategories:  # TODO: is this used? Whould not it be better to have Subject vocabulary?
```

Comment:
In rdm, this is `subjects`

Reply:
in nma, we probably need to keep it separate, as we probably will not have dictionaries in the required form.